codecov.yml

```diff
@@ -9,14 +9,14 @@ coverage:
   status:
     project:
       frontend:
-        threshold: 0%
+        threshold: 0.02%
         flags: frontend
       backend:
-        threshold: 0%
+        threshold: 0.02%
```

Comment:
We keep getting a lot of 0.01% coverage drops when code is removed. To keep the angry red :x: from marring perfectly good PRs, we'll try a slightly non-zero threshold and see how that works.